azure-pipelines-slow.yml

```diff
@@ -0,0 +1,67 @@
+# CI Trigger on main branch
+trigger:
+  branches:
+    include:
+    - main
+    - release/3.1.4xx
+  paths:
+    include:
+    - eng/Version.Details.xml
+
+# Trigger builds for PR's targeting main
+pr:
+  branches:
+    include:
+    - main
+  paths:
+    exclude: # don't trigger the CI if only a doc file was changed
+    - docs/*
+    - eng/common/README.md
+    - src/benchmarks/micro/Serializers/README.md
+    - src/benchmarks/micro/README.md
+    - src/benchmarks/real-world/JitBench/README.md
+    - src/benchmarks/real-world/Microsoft.ML.Benchmarks/Input/README.md
+    - src/tools/ResultsComparer/README.md
+    - README.md
+
+schedules:
+- cron: "0 */12 * * *"
+  displayName: Every 12 hours build
+  branches:
+    include:
+    - main
```

Comment:
We should remove this from triggering on main. We have coverage of Arm64 on the runtime side, and given how little Arm64 capacity we have we should only run what we have to here, the 3.1 runs.